src/cascadia/TerminalCore/Terminal.hpp

```diff
@@ -131,8 +131,7 @@ class Microsoft::Terminal::Core::Terminal final :
     // These methods are defined in TerminalApi.cpp
     void ReturnResponse(const std::wstring_view response) override;
     Microsoft::Console::VirtualTerminal::StateMachine& GetStateMachine() noexcept override;
-    TextBuffer& GetTextBuffer() noexcept override;
-    til::rect GetViewport() const noexcept override;
+    std::tuple<TextBuffer&, til::rect, bool> GetBufferAndViewport() noexcept override;

```

Comment:
I believe returning a `struct` with 3 named members will make the code more readable. Something like `std::get<bool>(GetBufferAndViewport())` is quite difficult to understand for newcomers after all. Additionally, a `struct` has the benefit that in an IDE like VS you can click the members and get a list of places the members is being used at. This would simplify code changes in the future.

Reply:
Yeah, this was just me being a bit lazy. I'll do that.